pkg/drivers/kic/oci/network.go

```diff
@@ -73,6 +78,17 @@ func dockerGatewayIP() (net.IP, error) {
 	return ip, nil
 }
 
+// containerGatewayIP gets the default gateway ip for the container
+func containerGatewayIP(ociBin, containerName string) (net.IP, error) {
+	rr, err := runCmd(exec.Command(ociBin, "inspect", "--format", "{{.NetworkSettings.Gateway}}", containerName))
+	if err != nil {
+		return nil, errors.Wrapf(err, "inspect gateway")
+	}
+	ip := net.ParseIP(strings.TrimSpace(rr.Stdout.String()))
+	glog.Infof("got host ip for mount in container by inspecting container: %s", ip.String())
```

Comment:
Is this log statement useful to keep?

If so: I don't think the `mount` noun is useful here. It also shouldn't be necessary to call `.String()` on the IP, as the formatter will do so.

Reply:
I think it was mostly copied from the docker functions, we can remove the logging

Reply:
yeah we can remove the logging